Lesson22/todo-app/Gokhan/script.js

```diff
@@ -0,0 +1,67 @@
+// const sayHelloBtn = document.getElementById('btnSayHello');
+
+// sayHelloBtn.addEventListener('cli('ck', function (e) {
+//   console.log('Hello', e);
+// });
+
+// const sayByeBtn = document.getElementById('btnSayBye');
+
+// sayByeBtn.addEventListener('click', function () {
+//   console.log('Bye bye');
+// });
+
+const form = document.getElementById('toDoForm');
+
+form.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+
+	// Usually you always need it for form submit events!
+
+	event.preventDefault();
+
+	const input = document.getElementById('toDoInput');
+
+	const listItem = document.createElement('li');
+	listItem.innerText = input.value;
+	listItem.addEventListener('click', function () {
+		listItem.classList.toggle('completed');
+	});
+	
+		// H E L P :)
+		// I could't do anything about cross. How can i do that?
+
+	// TODO: Make sure that only the text of todo item is crossed
+
+	const button = document.createElement('button');
+	button.innerHTML = '&#128465';
+	button.style.fontSize = '16px';
```

Comment:
Add all the styles below in CSS as a class (e.g. button-delete). Then, only add that class for the button element here:button.classList.add('button-delete');